include/simple_blob_detector.hpp

```diff
@@ -0,0 +1,16 @@
+#ifndef __THEA_SIMPLE_BLOB_DETECTOR_H__
```

Comment:
You may replace `THEA` with `HASKELL_OPENCV`. Thea was the original name of the project. We still have to rename a few instances. Mainly in the header file guards.